templates/base.html

```diff
@@ -1,6 +1,6 @@
 {%- import 'macros.html' as macros -%}
 <!DOCTYPE html>
-<html class="no-js" lang="en">
+<html class="no-js" lang="en" prefix="og:https://sampleplatform.ccextractor.org.me/">
```

Comment:
https://stackoverflow.com/questions/27943451/meaning-of-html-prefix-attribute-open-graph-protocol

There needs to be a space between `og:` and the url. Also, it would be nice if you go through the entire answer :smile: